apps/examples/src/index.js

```diff
@@ -7,9 +7,7 @@
 
 import { LogBox } from 'react-native';
 
-// Taken from "npm:expo/AppEntry.js" to fix broken "package.main"
-// in Expo template when used in npm workspaces
-import registerRootComponent from 'expo/build/launch/registerRootComponent';
+import { registerRootComponent } from 'expo';
```

Comment:
Based on https://docs.expo.dev/guides/monorepos/#change-default-entrypoint

Reply:
This fixes:
![Screenshot 2024-11-12 at 8 03 45 PM](https://github.com/user-attachments/assets/7cf6bed2-bbe2-4949-8380-90fc000eb1a1)

Reply:
This also fixes the error: 
```js
import registerRootComponent from 'expo/src/launch/registerRootComponent';
```